<!doctype html>
<html lang="en">

<head>

  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- jQuery UI css -->
  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">

  <!-- Google font, Roboto -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,300;0,400;0,700;1,400&family=Roboto:ital,wght@0,300;0,400;0,700;1,400&display=swap" rel="stylesheet">

  <!-- FontAwesome icon include -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />

  <!-- Main local styles -->
  <link rel="stylesheet" href="css/main.css">

  <title>Lukas M. Weber</title>

</head>

<body id="top">

  <!-- navbar include -->
  <div id="page-navbar"></div>

  <!-- main wrapper div -->
  <div class="container px-4" id="wrapper">

    <div class="row mx-4">

      <div class="col-sm-12">
        <h1 id="page-lead">Presentations</h1>

        <!-- Talks -->

        <h3><span>Talks</span></h3>

        <h4><span>Upcoming</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>University of Alabama at Birmingham, Department of Biostatistics Seminar Series, Birmingham, AL, Feb 2025.
            </li>
            <li>ENAR (Eastern North American Region of the International Biometric Society) 2025, New Orleans, LA, Mar 2025. Presentation in session on <i>Advanced statistical methods for integrative analysis of spatially resolved transcriptomics data.</i>
            </li>
          </ul>
        </div>

        <h4><span>2024</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Personalized Health 2024, ETH Zurich, Zurich, Switzerland. (2024). <i>Methods for scalable R-based analysis workflows for spatial transcriptomics data</i>
            </li>
            <li>University of Zurich, Zurich Seminars in Bioinformatics, Zurich, Switzerland. (2024). <i>Spatially variable genes: Methods, benchmarking, and future directions</i>
            </li>
            <li>BioC2024: Bioconductor Conference, Grand Rapids, MI, United States. (2024). <i>smoothclust: Identification of spatial domains by smoothing for compositional analyses in spatial transcriptomics data</i>
            </li>
            <li>NESS 2024: New England Statistics Symposium, Storrs, CT, United States. (2024). <i>smoothclust: Identification of spatial domains by spatial smoothing in spatial transcriptomics data</i>
            </li>
            <li>STATGEN 2024: Conference on Statistics in Genomics and Genetics, Pittsburgh, PA, United States. (2024). <i>smoothclust: Identification of spatial domains by spatial smoothing in spatial transcriptomics data</i>
            </li>
          </ul>
        </div>

        <h4><span>2023</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>University of Southern California, Division of Biostatistics, Seminar Series, Los Angeles, CA, United States. (2023). <i>nnSVG for the identification of spatially variable genes and unsupervised analyses of spatial transcriptomics data.</i>
            </li>
            <li>Columbia University, Department of Biostatistics, Statistical Genomics and Genetics Seminar Series. (Virtual). (2023). <i>nnSVG for the identification of spatially variable genes and unsupervised analyses of spatial transcriptomics data.</i>
            </li>
            <li>Boston University, Chobanian and Avedisian School of Medicine, Section on Computational Biomedicine, Spatial Biology Seminar Series. (Virtual). (2023). <i>nnSVG for the identification of spatially variable genes and unsupervised analyses of spatial transcriptomics data.</i>
            </li>
            <li>Joint Statistical Meetings (JSM) 2023, Toronto, Canada. (2023). <i>nnSVG for preprocessing, feature selection, and quality control in multi-sample spot-based spatially-resolved transcriptomics data.</i>
            </li>
            <li>Statistical Methods in Imaging Conference 2023, Annual Meeting of the ASA Statistics in Imaging Section, Minneapolis, MN, United States. (2023). <i>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes.</i>
            </li>
            <li>Emerging Leaders in Computational Oncology 2023, Memorial Sloan Kettering Cancer Center, New York, NY, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>Boston University, School of Public Health, Department of Biostatistics, Boston, MA, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>Ohio State University, College of Medicine, Department of Biomedical Informatics, Columbus, OH, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>University of Colorado, School of Medicine, Department of Biomedical Informatics. (Virtual). (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>Emory University, Rollins School of Public Health, Department of Biostatistics and Bioinformatics, Atlanta, GA, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>University of Utah, School of Medicine, Department of Human Genetics, Salt Lake City, UT, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>University of Minnesota, School of Public Health, Division of Biostatistics, Minneapolis, MN, United States. (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
            <li>Northwestern University, Feinberg School of Medicine, Department of Cell and Developmental Biology. (Virtual). (2023). <i>Unsupervised statistical methods and data-driven analysis workflows for spatially-resolved transcriptomics.</i>
            </li>
          </ul>
        </div>

        <h4><span>2022</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>European Conference on Computational Biology (ECCB), Workshop NTB-W04. (Virtual). (2022). <i>Scalable identification of spatially variable genes with nnSVG and Bioconductor.</i>
            </li>
            <li>R/Medicine. (Virtual). (2022). <i>Unsupervised analyses of spatially-resolved transcriptomics data with nnSVG and R/Bioconductor.</i>
            </li>
            <li>Sydney Bioinformatics Seminar Series, Sydney Precision Bioinformatics Alliance, University of Sydney. (Virtual). (2022). <i>nnSVG: scalable identification of spatially variable genes in spatially-resolved transcriptomics data.</i>
            </li>
            <li>Bioc2022 Bioconductor annual conference, Seattle, WA, United States. (2022). <i>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes.</i>
            </li>
            <li>BLAST Working Group seminar, Department of Biostatistics, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD, United States. (2022). <i>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes.</i>
            </li>
            <li>ENAR (Eastern North American Region of the International Biometric Society) Spring Meeting, Houston, TX, United States. (2022). <i>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes.</i>
            </li>
          </ul>
        </div>

        <h4><span>2021</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Bioc2021 Bioconductor annual conference. (Virtual). (2021). <i>Workshop on 'Orchestrating Spatially-Resolved Transcriptomics Analysis with Bioconductor (OSTA)'</i>
            </li>
            <li>Bioc2021 Bioconductor annual conference. (Virtual). (2021). <i>Workshop on 'SpatialExperiment'</i> (Joint presentation with Dario Righelli and Helena L. Crowell.)
            </li>
          </ul>
        </div>

        <h4><span>2020</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>European Bioconductor Meeting 2020. (Virtual). (2020). <i>Workshop on 'SpatialExperiment'.</i> (Joint presentation with Dario Righelli and Helena L. Crowell.)
            </li>
            <li>Bioc2020 Bioconductor annual conference. (Virtual). (2020). <i>Unsupervised analysis of transcriptome-scale spatial gene expression data in the human prefrontal cortex.</i>
            </li>
          </ul>
        </div>

        <h4><span>Earlier years</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Johns Hopkins University 13th Annual Genomics and Bioinformatics Symposium and Poster Session, Baltimore, MD, United States. (2019). <i>Comparison of dimension reduction algorithms for visualization of single-cell data.</i>
            </li>
            <li>European Bioconductor Meeting 2018, Munich, Germany. (2018). <i>HDCytoData package: High-dimensional cytometry benchmark datasets in Bioconductor formats.</i>
            </li>
            <li>ICCABS 2018 (IEEE 8th International Conference on Computational Advances in Bio and Medical Sciences): 1st Workshop on Computational Advances for Single-Cell Omics Data Analysis (CASCODA), Las Vegas, NV, United States. (2018). <i>Methods, tools, and resources for differential discovery in high-dimensional cytometry data.</i>
            </li>
            <li>1st Swiss Cytometry Meeting, Lausanne, Switzerland. (2018). <i>Statistical methods for differential discovery in high-dimensional cytometry data.</i>
            </li>
            <li>European Bioconductor Meeting 2017, Cambridge, United Kingdom. (2017). <i>Statistical methods for differential discovery in high-dimensional cytometry data.</i>
            </li>
            <li>CYTO 2017: 32nd Congress of the International Society for Advancement of Cytometry, Boston, MA, United States. (2017). <i>Statistical methods for differential discovery in high-dimensional cytometry data.</i>
            </li>
            <li>IMLS Scientific Retreat 2017 (Institute of Molecular Life Sciences, University of Zurich), Emmetten, Switzerland. (2017). <i>Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry (CyTOF) data.</i>
            </li>
            <li>European Bioconductor Developers' Meeting 2015, Cambridge, United Kingdom. (2015). <i>regsplice: Lasso-based model selection for improved detection of differential exon usage.</i>
            </li>
            <li>C1omics 2015: Single-Cell Omics Methods and Applications, Manchester, United Kingdom. (2015). <i>Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data.</i>
            </li>
          </ul>
        </div>

        <!-- Posters -->

        <h3><span>Posters</span></h3>

        <h4><span>2023</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>American College of Neuropsychopharmacology (ACNP) 2023, Tampa, FL, United States. (2023). <a href="https://zenodo.org/records/10291239"><i>nnSVG for the identification of spatially variable genes and improved computational analyses of spatial transcriptomic data from postmortem brain tissue</i></a>.
            </li>
            <li>National Human Genome Research Institute (NHGRI) Research Training and Career Development Annual Meeting 2023, Salt Lake City, UT, United States. (2023). <a href="https://zenodo.org/record/7814740"><i>nnSVG: scalable identification of spatially variable genes using nearest-neighbor Gaussian processes</i></a>.
            </li>
          </ul>
        </div>

        <h4><span>2022</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>American College of Neuropsychopharmacology (ACNP) 2022, Phoenix, AZ, United States. (2022). (Travel awardee.) <a href="https://zenodo.org/record/7448748"><i>The gene expression landscape of the human locus coeruleus revealed by single-nucleus and spatially-resolved transcriptomics</i></a>.
            </li>
          </ul>
        </div>

        <h4><span>2020</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>ASHG Institute for Genomic Medicine Symposium on Spatial Omics. (Virtual). (2020). <a href="https://zenodo.org/record/4110719"><i>Unsupervised analysis of transcriptome-scale spatial gene expression in human dorsolateral prefrontal cortex using spatial transcriptomics data</i></a>.
            </li>
            <li>Genome Informatics. (Poster and lightning talk.) (Virtual). (2020). <a href="https://zenodo.org/record/4110719"><i>Unsupervised analysis of transcriptome-scale spatial gene expression in human dorsolateral prefrontal cortex using spatial transcriptomics data</i></a>.
            </li>
          </ul>
        </div>

        <h4><span>Earlier years</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>CYTO 2018: 33rd Congress of the International Society for Advancement of Cytometry, Prague, Czechia. (2018). <i>diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering.</i>
            </li>
            <li>Ascona Workshop 2017: Statistical Challenges in Single-Cell Biology, Ascona, Switzerland. (2017) <i>Statistical methods for differential discovery in high-dimensional cytometry data.</i>
            </li>
            <li>ICML 2016 (International Conference on Machine Learning): Workshop on Computational Biology, New York, NY, United States. (2016). <i>Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry (CyTOF) data.</i>
            </li>
            <li>SIB Days: Swiss Institute of Bioinformatics annual conference, Biel/Bienne, Switzerland. (2016). <i>Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry (CyTOF) data.</i>
            </li>
            <li>IMLS Scientific Retreat 2015 (Institute of Molecular Life Sciences, University of Zurich), Morschach, Switzerland. (2015). <i>Improving power to detect differential exon usage by L1-regularization (lasso) model selection.</i>
            </li>
          </ul>
        </div>

      </div>
      <!-- col-sm-12 -->
    </div>
    <!-- .row -->

  </div>
  <!-- #wrapper .container -->

  <!-- Footer include -->
  <div id="page-footer"></div>

  <!-- jquery JS minified -->
  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>

  <!-- jquery ui -->
  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

  <!-- popper minified -->
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>

  <!-- bootstrap JS minified -->
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>

  <!-- Main local scripts -->
  <script src="js/main.js"></script>

  <script>
    $(function() {

      // Load the top page navbar
      $("#page-navbar").load("page-navbar.html", function() {

        $("#about").removeClass("active");
        $("#presentations").addClass("active");

      });

    });
    // document.ready
  </script>

</body>

</html>
