<!doctype html>
<html lang="en">

<head>

  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- Favicon -->
  <link rel="apple-touch-icon" sizes="57x57" href="img/favicons/apple-icon-57x57.png">
  <link rel="apple-touch-icon" sizes="60x60" href="img/favicons/apple-icon-60x60.png">
  <link rel="apple-touch-icon" sizes="72x72" href="img/favicons/apple-icon-72x72.png">
  <link rel="apple-touch-icon" sizes="76x76" href="img/favicons/apple-icon-76x76.png">
  <link rel="apple-touch-icon" sizes="114x114" href="img/favicons/apple-icon-114x114.png">
  <link rel="apple-touch-icon" sizes="120x120" href="img/favicons/apple-icon-120x120.png">
  <link rel="apple-touch-icon" sizes="144x144" href="img/favicons/apple-icon-144x144.png">
  <link rel="apple-touch-icon" sizes="152x152" href="img/favicons/apple-icon-152x152.png">
  <link rel="apple-touch-icon" sizes="180x180" href="img/favicons/apple-icon-180x180.png">
  <link rel="icon" type="image/png" sizes="192x192" href="img/favicons/android-icon-192x192.png">
  <link rel="icon" type="image/png" sizes="32x32" href="img/favicons/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="96x96" href="img/favicons/favicon-96x96.png">
  <link rel="icon" type="image/png" sizes="16x16" href="img/favicons/favicon-16x16.png">
  <link rel="manifest" href="img/favicons/manifest.json">
  <meta name="msapplication-TileColor" content="#ffffff">
  <meta name="msapplication-TileImage" content="img/favicons/ms-icon-144x144.png">
  <meta name="theme-color" content="#ffffff">

  <!-- jQuery UI css -->
  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">

  <!-- Google font, Roboto -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,300;0,400;0,700;1,400&family=Roboto:ital,wght@0,300;0,400;0,700;1,400&display=swap" rel="stylesheet">


  <!-- FontAwesome icon include -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />

  <!-- Main local styles -->
  <link rel="stylesheet" href="css/main.css">

  <title>Lukas M. Weber</title>

</head>

<body id="top">

  <!-- navbar include -->
  <div id="page-navbar"></div>

  <!-- main wrapper div -->
  <div class="container px-4" id="wrapper">

    <div class="row mx-4">

      <div class="col-sm-12">
        <h1 id="page-lead">Publications</h1>

        <!-- Google Scholar -->

        <h3><span>Google Scholar</span></h3>

        <p>A complete list of publications is also available from <a href="Google Scholar">https://scholar.google.com/citations?user=E9G88AQAAAAJ&hl=en</a>.</p>

        <!-- Preprints -->

        <h3><span>Preprints</span></h3>

        <div class="col-sm-12">
          <ul id="preprints-list">
            <li>Hippen A.A., Davidson N.R., Barnard M.E., <b>Weber L.M.</b>, Gertz J., Doherty J.A., Hicks S.C., and Greene C.S. (2023). <i>Deconvolution reveals compositional differences in high-grade serous ovarian cancer subtypes.</i> bioRxiv (preprint). <a href="Link">https://www.biorxiv.org/content/10.1101/2023.06.14.544991v1</a>.
            </li>
          </ul>
        </div>

        <!-- Journal articles (accepted or in press) -->

        <h3><span>Journal articles (accepted or in press)</span></h3>

        <!-- Journal articles (peer-reviewed and published) -->

        <h3><span>Journal articles (peer-reviewed and published)</span></h3>

        <div class="col-sm-12">
          <ul id="published-list">
            <li><b>Weber L.M.*</b>, Divecha H.R.*, Tran M.N., Kwon S.H., Spangler A., Montgomery K.D., Tippani M., Bharadwaj R., Kleinman J.E., Page S.C., Hyde T.M., Collado-Torres L., Maynard K.R., Martinowich K.*, and Hicks S.C.* (2023). <i>The gene expression landscape of the human locus coeruleus revealed by single-nucleus and spatially-resolved transcriptomics.</i> eLife, 12:RP84628. <a href="Link">https://elifesciences.org/articles/84628</a>.
            </li>
            <li>Tippani M., Divecha H.R., Catallini II J.L., Kwon S.H., <b>Weber L.M.</b>, Spangler A., Jaffe A.E., Hyde T.M., Kleinman J.E., Hicks S.C., Martinowich K., Collado-Torres L., Page S.C., and Maynard K.R. (2023). <i>VistoSeg: Processing utilities for high-resolution images for spatially resolved transcriptomics data.</i> Biological Imaging, 3, e23. <a href="Link">https://www.cambridge.org/core/journals/biological-imaging/article/vistoseg-processing-utilities-for-highresolution-images-for-spatially-resolved-transcriptomics-data/990CBC4AC069F5EDC62316919398404B</a>.
            </li>
            <li>Hippen A.A., Omran D.K., <b>Weber L.M.</b>, Jung E., Drapkin R., Doherty J.A., Hicks S.C., and Greene C.S. (2023). <i>Performance of computational algorithms to deconvolve heterogeneous bulk ovarian tumor tissue depends on experimental factors.</i> Genome Biology, 24, 239. <a href="Link">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03077-7</a>.
            </li>
            <li><b>Weber L.M.</b>, Saha A., Datta A., Hansen K.D., and Hicks S.C. (2023). <i>nnSVG for the scalable identification of spatially variable genes using nearest-neighbor Gaussian processes.</i> Nature Communications, 14, 4059. (Note: Article featured in "Editors' Highlights for Biotechnology and Methods".) <a href="Link">https://www.nature.com/articles/s41467-023-39748-z</a>.
            </li>
            <li>Tiberi S., Crowell H.L., Samartsidis P., <b>Weber L.M.</b>, and Robinson M.D. (2023). <i>distinct: a novel approach to differential distribution analyses.</i> Annals of Applied Statistics, 17, 2, 1681--1700. <a href="Link">https://projecteuclid.org/journals/annals-of-applied-statistics/volume-17/issue-2/distinct-A-novel-approach-to-differential-distribution-analyses/10.1214/22-AOAS1689.full</a>.
            </li>
            <li>Krieg C., <b>Weber L.M.</b>, Fosso B., Hardiman G., Mileti E., El Aidy S., Marzano M., Robinson M.D., and Guglietta S. (2022). <i>Complement downregulation promotes an inflammatory signature that renders colorectal cancer susceptible to immunotherapy.</i> Journal for ImmunoTherapy of Cancer, 10, e004717. <a href="Link">https://jitc.bmj.com/content/10/9/e004717</a>.
            </li>
            <li>Pardo B., Spangler A., <b>Weber L.M.</b>, Hicks S.C., Jaffe A.E., Martinowich K., Maynard K.R., and Collado-Torres L. (2022). <i>spatialLIBD: an R/Bioconductor package to visualize spatially-resolved transcriptomics data.</i> BMC Genomics, 23, 434. <a href="Link">https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-022-08601-w</a>.
            </li>
            <li>Righell D.*, <b>Weber L.M.*</b>, Crowell H.L.*, Pardo B., Collado-Torres L., Ghazanfar S., Lun A.T.L., Hicks S.C.*, and Risso D.* (2022). <i>SpatialExperiment: infrastructure for spatially-resolved transcriptomics data in R using Bioconductor.</i> Bioinformatics, 38(11), 3128-3131. <a href="Link">https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac299/6575443</a>.
            </li>
            <li>Petrillo M., Fabbri M., Kagkli D.M. et al. (2022). <i>A roadmap for the generation of benchmarking resources for antimicrobial resistance detection using next generation sequencing.</i> F1000Research, v2, 10, 80. <a href="Link">https://f1000research.com/articles/10-80</a>.
            </li>
            <li><b>Weber L.M.</b>, Hippen A.A., Hickey P.F., Berrett K.C., Gertz J., Doherty J.A., Greene C.S., and Hicks S.C. (2021). <i>Genetic demultiplexing of pooled single-cell RNA-sequencing samples in cancer facilitates effective experimental design.</i> GigaScience, 10, 9, giab062. <a href="Link">https://academic.oup.com/gigascience/article/10/9/giab062/6374209</a>.
            </li>
            <li>Liechti T., <b>Weber L.M.</b>, Ashhurst T.M., Stanley N., Prlic M., Van Gassen S., and Mair F. (2021). <i>An updated guide for the perplexed: cytometry in the high-dimensional era.</i> Nature Immunology, 22, 1190--1197. <a href="Link">https://www.nature.com/articles/s41590-021-01006-z</a>.
            </li>
            <li>Hippen A.A., Falco M.M., <b>Weber L.M.</b>, Erkan E.P., Zhang K., Doherty J.A., Vähärautio A., Greene C.S., and Hicks S.C. (2021). <i>miQC: An adaptive probabilistic framework for quality control of single-cell RNA-sequencing data.</i> PLOS Computational Biology, 17, 8, e1009290. <a href="Link">https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009290</a>.
            </li>
            <li>Maynard K.R.*, Collado-Torres L.*, <b>Weber L.M.</b>, Uytingco C., Barry B.K., Williams S.R., Catallini J.L. II, Tran M.N., Besich Z., Tippani M., Chew J., Yin Y., Kleinman J.E., Hyde T.M., Rao N., Hicks S.C., Martinowich K.*, Jaffe A.E.* (2021). <i>Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex.</i> Nature Neuroscience, 24, 425--436. <a href="Link">https://www.nature.com/articles/s41593-020-00787-0</a>.
            </li>
            <li>Shaul M.E., Eyal O., Guglietta S., Aloni P., Zlotnik A., Forkosh E., Levy L., <b>Weber L.M.</b>, Levin Y., Pomerantz A., Nechushtan H., Eruslanov E., Singhal S., Robinson M.D., Krieg C., and Fridlender Z.G. (2020). <i>Circulating neutrophil subsets in advanced lung cancer patients exhibit unique immune signature and relate to prognosis.</i> FASEB Journal, 34, 3, 4204--4218. <a href="Link">https://onlinelibrary.wiley.com/doi/full/10.1096/fj.201902467R</a>.
            </li>
            <li><b>Weber L.M.</b>, Saelens W., Cannoodt R., Soneson C., Hapfelmeier A., Gardner P.P., Boulesteix A.-L., Saeys Y., and Robinson M.D. (2019). <i>Essential guidelines for computational method benchmarking.</i> Genome Biology, 20, 125. <a href="Link">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8</a>.
            </li>
            <li><b>Weber L.M.</b> and Soneson C. (2019). <i>HDCytoData: Collection of high-dimensional cytometry benchmark datasets in Bioconductor object formats.</i> F1000Research, 8, 1459, v2. <a href="Link">https://f1000research.com/articles/8-1459</a>.
            </li>
            <li><b>Weber L.M.</b>, Nowicka N., Soneson C., and Robinson M.D. (2019). <i>diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering.</i> Communications Biology, 2, 183. <a href="Link">https://www.nature.com/articles/s42003-019-0415-5</a>.
            </li>
            <li>Krieg C., Nowicka M., Guglietta S., Schindler S., Hartmann F.J., <b>Weber L.M.</b>, Dummer R., Robinson M.D., Levesque M.P.* and Becher B.* (2018). <i>High-dimensional single-cell analysis predicts response to anti-PD-1 immunotherapy.</i> Nature Medicine, 24, 2, 144--153. <a href="Link">https://www.nature.com/articles/nm.4466</a>.
            </li>
            <li>Nowicka N., Krieg C., Crowell H.L., <b>Weber L.M.</b>, Hartmann F.J., Guglietta S., Becher B., Levesque M.P., and Robinson M.D. (2019). <i>CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets.</i> F1000Research, 6, 748, v4. <a href="Link">https://f1000research.com/articles/6-748</a>.
            </li>
            <li><b>Weber L.M.</b> and Robinson M.D. (2016). <i>Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data.</i> Cytometry Part A, 89A, 1084--1096. (Note: Selected as an "Editor's Choice" article.) <a href="Link">https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030</a>.
            </li>
            <li>Hartmann F.J., Bernard-Valnet R., Quériault C., Mrdjen D., <b>Weber L.M.</b>, Galli E., Krieg C., Robinson M.D., Nguyen X.H., Dauvilliers Y., Liblau R.S., and Becher B. (2016). <i>High-dimensional single-cell analysis reveals the immune signature of narcolepsy.</i> Journal of Experimental Medicine, 213, 12, 2621--2633. <a href="Link">http://jem.rupress.org/content/213/12/2621</a>.
            </li>
            <li>Burger A., Lindsay H., Felker A., Hess C., Anders C., Chiavacci E., Zaugg J., <b>Weber L.M.</b>, Catena R., Jinek M., Robinson M.D., and Mosimann C. (2016). <i>Maximizing mutagenesis with solubilized CRISPR-Cas9 ribonucleoprotein complexes.</i> Development, 143, 2025--2037. <a href="Link">https://dev.biologists.org/content/143/11/2025</a>.
            </li>
            <li>Jackson K., <b>Weber L.M.</b>, and Tennant M. (2016). <i>A retrospective study of the effectiveness of four different treatments of periodontal disease in equine cheek teeth.</i> Journal of Veterinary Dentistry, 33, 2, 83--89. <a href="Link">https://journals.sagepub.com/doi/10.1177/0898756416661624</a>.
            </li>
            <li>Robinson M.D., Kahraman A., Law C.W., Lindsay H., Nowicka M., <b>Weber L.M.</b>, and Zhou X. (2014). <i>Statistical methods for detecting differentially methylated loci and regions.</i> Frontiers in Genetics, 5, 324, 1--7. <a href="Link">https://www.frontiersin.org/articles/10.3389/fgene.2014.00324</a>.
            </li>
          </ul>
        </div>

      </div>
      <!-- col-sm-12 -->
    </div>
    <!-- .row -->

  </div>
  <!-- #wrapper .container -->

  <!-- jquery JS minified -->
  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>

  <!-- jquery ui -->
  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

  <!-- popper minified -->
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>

  <!-- bootstrap JS minified -->
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>

  <!-- Main local scripts -->
  <script src="js/main.js"></script>

  <script>
    $(function() {

      // Load the top page navbar
      $("#page-navbar").load("page-navbar.html", function() {

        $("#me").removeClass("active");
        $("#research").addClass("active");

      });

    });
    // document.ready
  </script>

</body>

</html>
