<!doctype html>
<html lang="en">

<head>

  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- jQuery UI css -->
  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">

  <!-- Google font, Roboto -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,300;0,400;0,700;1,400&family=Roboto:ital,wght@0,300;0,400;0,700;1,400&display=swap" rel="stylesheet">

  <!-- FontAwesome icon include -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />

  <!-- Main local styles -->
  <link rel="stylesheet" href="css/main.css">

  <title>Lukas M. Weber</title>

</head>

<body id="top">

  <!-- navbar include -->
  <div id="page-navbar"></div>

  <!-- main wrapper div -->
  <div class="container px-4" id="wrapper">

    <div class="row mx-4">

      <div class="col-sm-12">
        <h1 id="page-lead">Research</h1>

        <p>Overview of recent research topics, along with links to key papers and other associated resources. For more details, see <a href="https://lmweber.org/publications.html">Publications</a> and <a href="https://lmweber.org/software.html">Software</a>.</p>

        <!-- Methods -->

        <h3><span>Methods for single-cell and spatial omics data</span></h3>

        <h4><span>Unsupervised statistical methods</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Development of a scalable method to identify spatially variable genes in spatial transcriptomics data: <i>nnSVG</i> (<a href="https://www.nature.com/articles/s41467-023-39748-z">Weber et al. 2023</a>) (<a href="https://bioconductor.org/packages/nnSVG">R/Bioconductor package</a>)
            </li>
          </ul>
        </div>

        <h4><span>Differential analyses</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Development of an improved computational framework for differential analyses in high-dimensional cytometry data: <i>diffcyt</i> (<a href="https://www.nature.com/articles/s42003-019-0415-5">Weber et al. 2019</a>) (<a href="https://bioconductor.org/packages/diffcyt">R/Bioconductor package</a>)
            </li>
          </ul>
        </div>

        <h4><span>Applications of spatial statistics</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li><i>nnSVG</i> (see 'Unsupervised statistical methods' above)
            </li>
          </ul>
        </div>

        <!-- Collaborative analyses -->

        <h3><span>Collaborative analyses of single-cell and spatial omics data</span></h3>

        <h4><span>Biological applications in neuroscience</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Collaborative analysis of single-nucleus and spatial transcriptomics data from the locus coeruleus region in postmortem human brain samples (<a href="https://elifesciences.org/articles/84628">Weber and Divecha et al. 2023</a>)
            </li>
            <li>Development of an unsupervised analysis workflow during a collaborative analysis of spatial transcriptomics data from the dorsolateral prefrontal cortex (DLPFC) region in postmortem human brain samples (<a href="https://www.nature.com/articles/s41593-020-00787-0">Maynard and Collado-Torres et al. 2021</a>)
            </li>
          </ul>
        </div>

        <h4><span>Applications in cancer biology</span></h4>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Benchmark evaluation of methods for genetic variation-based demultiplexing of pooled single-cell RNA sequencing samples from tumor samples from high-grade serous ovarian cancer (HGSOC) and lung adenocarcinoma (<a href="https://academic.oup.com/gigascience/article/10/9/giab062/6374209">Weber et al. 2021</a>)
            </li>
          </ul>
        </div>

        <!-- Benchmarking -->

        <h3><span>Benchmarking studies</span></h3>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Benchmark of methods for genetic variation-based demultiplexing of pooled single-cell RNA sequencing samples from tumors (see 'Applications in cancer biology' above)
            </li>
            <li>Review and best practices guidelines on performing different types of benchmarking studies (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8">Weber et al. 2019</a>)
            </li>
            <li>Benchmark comparison of clustering methods for high-dimensional cytometry data (<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030">Weber and Robinson 2016</a>)
            </li>
          </ul>
        </div>

        <!-- Analysis workflows -->

        <h3><span>Analysis workflows and software infrastructure</span></h3>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Interactive online textbook demonstrating best practices for analysis workflows for spatial transcriptomics data within the R/Bioconductor framework, including example R code and datasets: <a href="https://lmweber.org/BestPracticesST/">Best Practices for Spatial Transcriptomics Analysis with Bioconductor</a>
            </li>
            <li>Software infrastructure for storing and manipulating spatial transcriptomics datasets within the R/Bioconductor framework: <i>SpatialExperiment</i> (<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac299/6575443">Righelli, Weber, Crowell et al. 2022</a>) (<a href="https://bioconductor.org/packages/SpatialExperiment">R/Bioconductor package</a>)
            </li>
            <li>Contributions to the development of a comprehensive analysis workflow for high-dimensional cytometry data (<a href="https://f1000research.com/articles/6-748">Nowicka et al. 2019</a>) (<a href="https://bioconductor.org/packages/cytofWorkflow">R/Bioconductor package</a>)
            </li>
          </ul>
        </div>

        <!-- Open science -->

        <h3><span>Open science</span></h3>

        <div class="col-sm-12">
          <ul id="dot-points">
            <li>Development of open-source R packages through the <a href="https://bioconductor.org/">Bioconductor</a> project (see <a href="https://lmweber.org/software.html">Software</a> for details)
            </li>
            <li>Publication of preprints (<a href="https://biorxiv.org/">bioRxiv</a> and <a href="https://arxiv.org/">arXiv</a>)
            </li>
            <li>Reproducible code and data resources (see <a href="https://lmweber.org/software.html">Software</a> and <a href="https://github.com/lmweber">GitHub</a> for details)
            </li>
            <li><a href="https://researchsymbionts.org/">Research Symbiont</a> award, received in January 2021 in recognition of efforts to create open code and data resources
            </li>
          </ul>
        </div>

      </div>
      <!-- col-sm-12 -->
    </div>
    <!-- .row -->

  </div>
  <!-- #wrapper .container -->

  <!-- Footer include -->
  <div id="page-footer"></div>

  <!-- jquery JS minified -->
  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>

  <!-- jquery ui -->
  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

  <!-- popper minified -->
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>

  <!-- bootstrap JS minified -->
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>

  <!-- Main local scripts -->
  <script src="js/main.js"></script>

  <script>
    $(function() {

      // Load the top page navbar
      $("#page-navbar").load("page-navbar.html", function() {

        $("#about").removeClass("active");
        $("#research").addClass("active");

      });

    });
    // document.ready
  </script>

</body>

</html>
