<!doctype html>
<html lang="en">

<head>

  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- jQuery UI css -->
  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">

  <!-- Bootstrap CSS -->
  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">

  <!-- Google font, Roboto -->
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Mulish:ital,wght@0,300;0,400;0,700;1,400&family=Roboto:ital,wght@0,300;0,400;0,700;1,400&display=swap" rel="stylesheet">


  <!-- FontAwesome icon include -->
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />

  <!-- Main local styles -->
  <link rel="stylesheet" href="css/main.css">

  <title>Lukas M. Weber</title>

</head>

<body id="top">

  <!-- navbar include -->
  <div id="page-navbar"></div>

  <!-- main wrapper div -->
  <div class="container px-4" id="wrapper">

    <div class="row mx-4">

      <div class="col-sm-12">
        <h1 id="page-lead">Software</h1>

        <!-- GitHub -->

        <h3><span>GitHub</span></h3>

        <p>Additional details are available from <a href="https://github.com/lmweber">GitHub</a>.</p>

        <!-- Software packages -->

        <h3><span>Software packages</span></h3>

        <div class="col-sm-12">
          <ul>
            <li><a href="https://bioconductor.org/packages/nnSVG">nnSVG</a>: R/Bioconductor package to identify spatially variable genes in spatially-resolved transcriptomics data (Weber et al. 2023).
            </li>
            <li><a href="https://bioconductor.org/packages/SpatialExperiment">SpatialExperiment</a>: R/Bioconductor data structure for storing spatially-resolved transcriptomics datasets (Righelli, Weber, Crowell et al. 2022). (Joint project with Dario Righelli and Helena L. Crowell.)
            </li>
            <li><a href="https://bioconductor.org/packages/ggspavis">ggspavis</a>: R/Bioconductor package containing visualization functions for spatially-resolved transcriptomics data. (Joint project with Helena L. Crowell.)
            </li>
            <li><a href="https://bioconductor.org/packages/diffcyt">diffcyt</a>: R/Bioconductor package implementing a computational framework for differential discovery analyses in high-dimensional cytometry data (Weber et al. 2019).
            </li>
            <li><a href="https://bioconductor.org/packages/regsplice">regsplice</a>: R/Bioconductor package implementing statistical methodology for detection of differential splicing (differential exon usage) in RNA sequencing data.
            </li>
          </ul>
        </div>

        <!-- Other software projects -->

        <h3><span>Other software projects</span></h3>

        <div class="col-sm-12">
          <ul>
            <li><a href="https://lmweber.org/BestPracticesST/">Best Practices for Spatial Transcriptomics Analysis with Bioconductor</a>: Online book containing interactive example analysis workflows for spatial transcriptomics data.
            </li>
            <li><a href="https://lmweber.org/CrispantCal/">CrispantCal</a>: R/Shiny web application to calculate injection mix volumes for CRISPR-Cas9 experiments (described in our publication: Burger et al. 2016).
            </li>
          </ul>
        </div>

        <!-- Data packages -->

        <h3><span>Data packages</span></h3>

        <div class="col-sm-12">
          <ul>
            <li><a href="https://bioconductor.org/packages/WeberDivechaLCdata">WeberDivechaLCdata</a>: R/Bioconductor package containing datasets from our paper on spatially-resolved transcriptomics and single-nucleus RNA sequencing data in the human locus coeruleus (Weber and Divecha et al. 2023).
            </li>
            <li><a href="https://bioconductor.org/packages/STexampleData">STexampleData</a>: R/Bioconductor package containing a collection of spatially-resolved transcriptomics datasets in SpatialExperiment Bioconductor format.
            </li>
            <li><a href="https://bioconductor.org/packages/HDCytoData">HDCytoData</a>: R/Bioconductor package containing a collection of publicly available high-dimensional cytometry benchmark datasets in Bioconductor object formats (Weber and Soneson, 2019).
            </li>
          </ul>
        </div>

        <!-- Selected code repositories -->

        <h3><span>Selected code repositories</span></h3>

        <div class="col-sm-12">
          <ul class="col-sm-12">
            <li><a href="https://github.com/lmweber/locus-c/">locus-c</a>: Code repository to reproduce analyses and figures in our paper on analyzing spatial and single-nucleus data from the locus coeruleus (LC) region in the human brain (Weber and Divecha et al. 2023).
            </li>
            <li><a href="https://github.com/lmweber/nnSVG-analyses">nnSVG-analyses</a>: Code repository to reproduce analyses and figures in our paper on the 'nnSVG' method (Weber et al. 2023).
            </li>
            <li><a href="https://github.com/lmweber/snp-dmx-cancer">snp-dmx-cancer</a>: Snakemake pipeline and code repository to reproduce analyses and figures in our paper evaluating genetic variation-based demultiplexing for single-cell RNA sequencing (Weber et al. 2021).
            </li>
            <li><a href="https://github.com/lmweber/diffcyt-evaluations">diffcyt-evaluations</a>: Code repository to reproduce analyses and figures in our paper introducing the 'diffcyt' framework (Weber et al. 2019).
            </li>
            <li><a href="https://github.com/lmweber/cytometry-clustering-comparison">cytometry-clustering-comparison</a>: Code repository to reproduce analyses and figures in our benchmark paper comparing clustering algorithms for high-dimensional cytometry data (Weber and Robinson, 2016).
            </li>
          </ul>
        </div>

        <!-- Selected data repositories -->

        <h3><span>Selected data repositories</span></h3>

        <div class="col-sm-12">
          <ul>
            <li><a href="http://flowrepository.org/id/FR-FCM-ZYL8">FlowRepository FR-FCM-ZYL8</a>: Data repository containing datasets used for benchmarking analyses in our paper introducing the 'diffcyt' framework (Weber et al. 2019). These datasets are also provided in an easily accessible format in the 'HDCytoData' R/Bioconductor package.
            </li>
            <li><a href="http://flowrepository.org/id/FR-FCM-ZZPH">FlowRepository FR-FCM-ZZPH</a>: Data repository containing datasets used for benchmarking analyses in our paper comparing clustering algorithms for high-dimensional cytometry data (Weber and Robinson, 2016). These datasets are also provided in an easily accessible format in the 'HDCytoData' R/Bioconductor package.
            </li>
          </ul>
        </div>

      </div>
      <!-- col-sm-12 -->
    </div>
    <!-- .row -->

  </div>
  <!-- #wrapper .container -->

  <!-- jquery JS minified -->
  <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>

  <!-- jquery ui -->
  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>

  <!-- popper minified -->
  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>

  <!-- bootstrap JS minified -->
  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>

  <!-- Main local scripts -->
  <script src="js/main.js"></script>

  <script>
    $(function() {

      // Load the top page navbar
      $("#page-navbar").load("page-navbar.html", function() {

        $("#about").removeClass("active");
        $("#software").addClass("active");

      });

    });
    // document.ready
  </script>

</body>

</html>
