---
title: Publications
layout: page
---


### Google Scholar

A complete list of publications is also available from [Google Scholar](https://scholar.google.com/citations?user=1vZo3toAAAAJ&hl=en).


---


### Preprint articles

- Righell D.\*, **Weber L.M.\***, Crowell H.L.\*, Pardo B., Collado-Torres L., Ghazanfar S., Lun A.T.L., Hicks S.C.\*, and Risso D.\* (2022). *SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor.* bioRxiv (preprint). [Paper](https://www.biorxiv.org/content/10.1101/2021.01.27.428431v3), [R/Bioconductor package](https://bioconductor.org/packages/SpatialExperiment).

- Tiberi S., Crowell H.L., Samartsidis P., **Weber L.M.**, and Robinson M.D. (2022), *distinct: a novel approach to differential distribution analyses.* bioRxiv (preprint). [Paper](https://www.biorxiv.org/content/10.1101/2020.11.24.394213v4), [R/Bioconductor package](https://bioconductor.org/packages/distinct).

- Petrillo M., Fabbri M., Kagkli D.M. et al. (2022). *A roadmap for the generation of benchmarking resources for antimicrobial resistance detection using next generation sequencing.* F1000Research, v2, 10, 80. [Paper](https://f1000research.com/articles/10-80).

- Tippani M., Divecha H.R., Catallini II J.L., **Weber L.M.**, Spangler A., Jaffe A.E., Hicks S.C., Martinowich K., Collado-Torres L., Page S.C., and Maynard K.R. (2021). *VistoSeg: a MATLAB pipeline to process, analyze and visualize high-resolution histology images for Visium spatial transcriptomics data.* bioRxiv (preprint). [Paper](https://www.biorxiv.org/content/10.1101/2021.08.04.452489v1), [Software](http://research.libd.org/VistoSeg/), [Code](https://github.com/LieberInstitute/VistoSeg).

- Pardo B., Spangler A., **Weber L.M.**, Hicks S.C., Jaffe A.E., Martinowich K., Maynard K.R., and Collado-Torres L. (2021), *spatialLIBD: an R/Bioconductor package to visualize spatially-resolved transcriptomics data.* bioRxiv (preprint). [Paper](https://www.biorxiv.org/content/10.1101/2021.04.29.440149v1), [R/Bioconductor package](http://bioconductor.org/packages/spatialLIBD).

- Krieg C., Carloni S., **Weber L.M.**, Fosso B., Hardiman G., Mileti E., El Aidy S., Marzano M., Pesole G., Asnicar F., Segata N., Robinson M.D., and Guglietta S. (2021). *Loss of C3aR induces immune infiltration and inflammatory microbiota in a new spontaneous model of colon cancer.* bioRxiv (preprint). [Paper](https://www.biorxiv.org/content/10.1101/2021.01.18.426963v2).


<!-- --- -->


<!-- ### Accepted articles (in press) -->


---


### Journal articles (peer-reviewed and published)

- **Weber L.M.**, Hippen A.A., Hickey P.F., Berrett K.C., Gertz J., Doherty J.A., Greene C.S., and Hicks S.C. (2021). *Genetic demultiplexing of pooled single-cell RNA-sequencing samples in cancer facilitates effective experimental design.* GigaScience, 10, 9, giab062. [Paper](https://academic.oup.com/gigascience/article/10/9/giab062/6374209), [Code](https://github.com/lmweber/snp-dmx-cancer), [Data](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156793), [Data (controlled access)](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002262.v1.p1).

- Liechti T., **Weber L.M.**, Ashhurst T.M., Stanley N., Prlic M., Van Gassen S., and Mair F. (2021). *An updated guide for the perplexed: cytometry in the high-dimensional era.* Nature Immunology, 22, 1190--1197. [Paper](https://www.nature.com/articles/s41590-021-01006-z).

- Hippen A.A., Falco M.M., **Weber L.M.**, Erkan E.P., Zhang K., Doherty J.A., Vähärautio A., Greene C.S., and Hicks S.C. (2021). *miQC: An adaptive probabilistic framework for quality control of single-cell RNA-sequencing data.* PLOS Computational Biology, 17, 8, e1009290. [Paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009290), [R/Bioconductor package](https://bioconductor.org/packages/miQC).

- Maynard K.R.\*, Collado-Torres L.\*, **Weber L.M.**, Uytingco C., Barry B.K., Williams S.R., Catallini J.L. II, Tran M.N., Besich Z., Tippani M., Chew J., Yin Y., Kleinman J.E., Hyde T.M., Rao N., Hicks S.C., Martinowich K.\*, Jaffe A.E.\* (2021). *Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex.* Nature Neuroscience, 24, 425--436. [Paper](https://www.nature.com/articles/s41593-020-00787-0), [Web application](http://spatial.libd.org/spatialLIBD/), [R/Bioconductor package](http://bioconductor.org/packages/spatialLIBD), [Code for paper](https://github.com/LieberInstitute/HumanPilot), [Code for web application](https://github.com/LieberInstitute/spatialLIBD).

- Shaul M.E., Eyal O., Guglietta S., Aloni P., Zlotnik A., Forkosh E., Levy L., **Weber L.M.**, Levin Y., Pomerantz A., Nechushtan H., Eruslanov E., Singhal S., Robinson M.D., Krieg C., and Fridlender Z.G. (2020). *Circulating neutrophil subsets in advanced lung cancer patients exhibit unique immune signature and relate to prognosis.* FASEB Journal, 34, 3, 4204--4218. [Paper](https://onlinelibrary.wiley.com/doi/full/10.1096/fj.201902467R).

- **Weber L.M.**, Saelens W., Cannoodt R., Soneson C., Hapfelmeier A., Gardner P.P., Boulesteix A.-L., Saeys Y., and Robinson M.D. (2019). *Essential guidelines for computational method benchmarking.* Genome Biology, 20, 125. [Paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1738-8).

- **Weber L.M.** and Soneson C. (2019). *HDCytoData: Collection of high-dimensional cytometry benchmark datasets in Bioconductor object formats.* F1000Research, 8, 1459, v2. [Paper](https://f1000research.com/articles/8-1459), [R/Bioconductor package](http://bioconductor.org/packages/HDCytoData).

- **Weber L.M.**, Nowicka N., Soneson C., and Robinson M.D. (2019). *diffcyt: Differential discovery in high-dimensional cytometry via high-resolution clustering.* Communications Biology, 2, 183. [Paper](https://www.nature.com/articles/s42003-019-0415-5), [R/Bioconductor package](http://bioconductor.org/packages/diffcyt), [Code](https://github.com/lmweber/diffcyt-evaluations), [Data](http://flowrepository.org/id/FR-FCM-ZYL8).

- Krieg C., Nowicka M., Guglietta S., Schindler S., Hartmann F.J., **Weber L.M.**, Dummer R., Robinson M.D., Levesque M.P.\* and Becher B.\* (2018). *High-dimensional single-cell analysis predicts response to anti-PD-1 immunotherapy.* Nature Medicine, 24, 2, 144--153. [Paper](https://www.nature.com/articles/nm.4466).

- Nowicka N., Krieg C., Crowell H.L., **Weber L.M.**, Hartmann F.J., Guglietta S., Becher B., Levesque M.P., and Robinson M.D. (2019). *CyTOF workflow: differential discovery in high-throughput high-dimensional cytometry datasets.* F1000Research, 6, 748, v4. [Paper](https://f1000research.com/articles/6-748), [R/Bioconductor package](https://bioconductor.org/packages/cytofWorkflow).

- **Weber L.M.** and Robinson M.D. (2016). *Comparison of clustering methods for high-dimensional single-cell flow and mass cytometry data.* Cytometry Part A, 89A, 1084--1096. (Note: Selected as an 'Editor's Choice' article.) [Paper](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030), [Code](https://github.com/lmweber/cytometry-clustering-comparison), [Data](http://flowrepository.org/id/FR-FCM-ZZPH).

- Hartmann F.J., Bernard-Valnet R., Quériault C., Mrdjen D., **Weber L.M.**, Galli E., Krieg C., Robinson M.D., Nguyen X.H., Dauvilliers Y., Liblau R.S., and Becher B. (2016). *High-dimensional single-cell analysis reveals the immune signature of narcolepsy.* Journal of Experimental Medicine, 213, 12, 2621--2633. [Paper](http://jem.rupress.org/content/213/12/2621).

- Burger A., Lindsay H., Felker A., Hess C., Anders C., Chiavacci E., Zaugg J., **Weber L.M.**, Catena R., Jinek M., Robinson M.D., and Mosimann C. (2016). *Maximizing mutagenesis with solubilized CRISPR-Cas9 ribonucleoprotein complexes.* Development, 143, 2025--2037. [Paper](https://dev.biologists.org/content/143/11/2025).

- Jackson K., **Weber L.M.**, and Tennant M. (2016). *A retrospective study of the effectiveness of four different treatments of periodontal disease in equine cheek teeth.* Journal of Veterinary Dentistry, 33, 2, 83--89. [Paper](https://journals.sagepub.com/doi/10.1177/0898756416661624).

- Robinson M.D., Kahraman A., Law C.W., Lindsay H., Nowicka M., **Weber L.M.**, and Zhou X. (2014). *Statistical methods for detecting differentially methylated loci and regions.* Frontiers in Genetics, 5, 324, 1--7. [Paper](https://www.frontiersin.org/articles/10.3389/fgene.2014.00324).

