---
title: Software
layout: page
---


### GitHub

Links to my software packages and other code repositories are also available from [GitHub](https://github.com/lmweber).


---


### Online book

- [Orchestrating Spatially Resolved Transcriptomics Analysis with Bioconductor (OSTA)](https://lmweber.org/OSTA-book/): Online textbook describing steps in a computational analysis pipeline for spatially resolved transcriptomics data, including R code and example datasets. (Joint project with [several contributors](https://lmweber.org/OSTA-book/contributors.html).)


---


### Software packages

- [SpatialExperiment](https://bioconductor.org/packages/SpatialExperiment): R/Bioconductor object class for storing spatially resolved transcriptomics datasets. (Joint project with [Dario Righelli](https://github.com/drighelli) and [Helena L. Crowell](https://helenalc.github.io/).)

- [ggspavis](https://github.com/lmweber/ggspavis): Visualization functions for spatially resolved transcriptomics data.

- [STexampleData](https://bioconductor.org/packages/STexampleData): Collection of spatially resolved transcriptomics datasets in SpatialExperiment Bioconductor format.

- [diffcyt](https://bioconductor.org/packages/diffcyt): R package implementing a computational framework for differential discovery analyses in high-dimensional cytometry data.

- [HDCytoData](http://bioconductor.org/packages/HDCytoData): R package containing a number of publicly available high-dimensional cytometry benchmark datasets in Bioconductor object formats.

- [regsplice](http://bioconductor.org/packages/regsplice): R package implementing statistical methodology for detection of differential splicing (differential exon usage) in RNA-seq data.


---


### Selected code repositories

- [snp-dmx-cancer](https://github.com/lmweber/snp-dmx-cancer): Snakemake pipeline and code repository to reproduce analyses in our paper evaluating genetic variation-based demultiplexing for single-cell RNA sequencing ([Weber et al. 2021](https://www.biorxiv.org/content/10.1101/2020.11.06.371963v3)).

- [diffcyt-evaluations](https://github.com/lmweber/diffcyt-evaluations): GitHub repository containing code to reproduce the benchmarking analyses in our paper introducing the *diffcyt* framework ([Weber et al. 2019](https://www.nature.com/articles/s42003-019-0415-5)).

- [cytometry-clustering-comparison](https://github.com/lmweber/cytometry-clustering-comparison): GitHub repository containing reproducible code from our benchmark paper comparing clustering algorithms for high-dimensional cytometry data ([Weber and Robinson, 2016](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030)).


---


### Selected data repositories

- [FlowRepository FR-FCM-ZYL8](http://flowrepository.org/id/FR-FCM-ZYL8): Data repository containing datasets used in the benchmarking analyses from our paper introducing the *diffcyt* framework ([Weber et al. 2019](https://www.nature.com/articles/s42003-019-0415-5)). These datasets are also provided in an easily accessible format in the [HDCytoData](http://bioconductor.org/packages/HDCytoData) Bioconductor package.

- [FlowRepository FR-FCM-ZZPH](http://flowrepository.org/id/FR-FCM-ZZPH): Data repository containing benchmark datasets from our paper comparing clustering algorithms for high-dimensional cytometry data ([Weber and Robinson, 2016](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23030)). These datasets are also provided in an easily accessible format in the [HDCytoData](http://bioconductor.org/packages/HDCytoData) Bioconductor package.

