---
title: Software
layout: page
---


### GitHub

Additional details are available from [GitHub](https://github.com/lmweber).


---


### Software packages

- [nnSVG](https://bioconductor.org/packages/nnSVG): R/Bioconductor package to identify spatially variable genes in spatially-resolved transcriptomics data.

- [SpatialExperiment](https://bioconductor.org/packages/SpatialExperiment): R/Bioconductor data structure for storing spatially-resolved transcriptomics datasets. (Joint project with Dario Righelli and Helena L. Crowell.)

- [ggspavis](https://bioconductor.org/packages/ggspavis): R/Bioconductor package containing visualization functions for spatially-resolved transcriptomics data. (Joint project with Helena L. Crowell.)

- [diffcyt](https://bioconductor.org/packages/diffcyt): R/Bioconductor package implementing a computational framework for differential discovery analyses in high-dimensional cytometry data.

- [regsplice](https://bioconductor.org/packages/regsplice): R/Bioconductor package implementing statistical methodology for detection of differential splicing (differential exon usage) in RNA-sequencing data.


---


### Other software projects

- [CrispantCal](https://lmweber.org/CrispantCal/): R/Shiny web application to calculate injection mix volumes for CRISPR-Cas9 experiments (described in our publication: Burger et al. 2016).


---


### Data packages

- [STexampleData](https://bioconductor.org/packages/STexampleData): R/Bioconductor package containing a collection of spatially-resolved transcriptomics datasets in SpatialExperiment Bioconductor format.

- [HDCytoData](https://bioconductor.org/packages/HDCytoData): R/Bioconductor package containing a collection of publicly available high-dimensional cytometry benchmark datasets in Bioconductor object formats.


---


### Selected code repositories

- [locus-c](https://github.com/lmweber/locus-c/): Code repository to reproduce analyses and figures in our paper on analyzing SRT and snRNA-seq data from the locus coeruleus (LC) region in the human brain (Weber and Divecha et al. 2022).

- [nnSVG-analyses](https://github.com/lmweber/nnSVG-analyses): Code repository to reproduce analyses and figures in our paper on the 'nnSVG' method (Weber et al. 2022).

- [snp-dmx-cancer](https://github.com/lmweber/snp-dmx-cancer): Snakemake pipeline and code repository to reproduce analyses and figures in our paper evaluating genetic variation-based demultiplexing for single-cell RNA sequencing (Weber et al. 2021).

- [diffcyt-evaluations](https://github.com/lmweber/diffcyt-evaluations): Code repository to reproduce analyses and figures in our paper introducing the 'diffcyt' framework (Weber et al. 2019).

- [cytometry-clustering-comparison](https://github.com/lmweber/cytometry-clustering-comparison): Code repository to reproduce analyses and figures in our benchmark paper comparing clustering algorithms for high-dimensional cytometry data (Weber and Robinson, 2016).


---


### Selected data repositories

- [FlowRepository FR-FCM-ZYL8](http://flowrepository.org/id/FR-FCM-ZYL8): Data repository containing datasets used for benchmarking analyses in our paper introducing the 'diffcyt' framework (Weber et al. 2019). These datasets are also provided in an easily accessible format in the 'HDCytoData' R/Bioconductor package.

- [FlowRepository FR-FCM-ZZPH](http://flowrepository.org/id/FR-FCM-ZZPH): Data repository containing datasets used for benchmarking analyses in our paper comparing clustering algorithms for high-dimensional cytometry data (Weber and Robinson, 2016). These datasets are also provided in an easily accessible format in the 'HDCytoData' R/Bioconductor package.

